playpen/dev-setup.sh

```diff
@@ -80,10 +80,14 @@ pushd pulp
 setfacl -m user:apache:rwx .
 popd; popd
 
+echo "Disabling strict mode for ssh"
+echo 'StrictModes no' | sudo tee -a /etc/ssh/sshd_config
+sudo systemctl restart sshd
 
 echo "Starting services, this may take a few minutes due to mongodb journal allocation"
 for s in qpidd goferd mongod; do
   sudo systemctl start $s
+  sudo systemctl enable $s
```

Comment:
In master, I've added this line just before the start. If you wouldn't mind, move it to the same place so we won't double enable it when this PR gets merged forward to master.